fdbclient/NativeAPI.actor.cpp

```diff
@@ -3124,7 +3124,7 @@ ACTOR Future<Void> readVersionBatcher( DatabaseContext *cx, FutureStream< std::p
 			Future<Void> batch =
 				incrementalBroadcast(
 					getConsistentReadVersion(cx, count, flags, std::move(debugID)),
-					std::vector< Promise<GetReadVersionReply> >(std::move(requests)), CLIENT_KNOBS->BROADCAST_BATCH_SIZE);
+					std::vector< Promise<GetReadVersionReply> >(std::move(requests)), CLIENT_KNOBS->BROADCAST_BATCH_SIZE, TaskPriority::DefaultPromiseEndpoint);
```

Comment:
I could use `TaskPriority::GetConsistentReadVersion` here like in the read version batcher, if that seems better.